utils/matchUtils.ts

```
@@ -60,13 +101,61 @@ export interface MatchData {
+ * @return An object containing the extracted home and away team names.
+ */
+export function extractTeamsFromESPNEvent(event: any): {
+  homeTeam: string;
+  awayTeam: string;
+} {
+  const result = { homeTeam: "", awayTeam: "" };
+
+  if (
+    event.competitions &&
+    event.competitions[0] &&
+    event.competitions[0].competitors
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)